cfme/tests/automate/test_git_import.py

```diff
@@ -78,3 +78,41 @@ def test_automate_git_domain_displayed_in_service(appliance, imported_domain, br
     view.modal.apply.click()
     assert view.provisioning_entry_point.value == ("/{}/Service/Provisioning/StateMachines/"
         "ServiceProvision_Template/CatalogItemInitialization".format(imported_domain.name))
+
+
+@pytest.mark.tier(3)
+@pytest.mark.parametrize(
+    ("url", "param_type", "param_value", "verify_ssl"),
+    [("https://github.com/ganeshhubale/ManageIQ-automate-git", "branch", "origin/master", True)],
+    ids=["multiple_domains"],
+)
+def test_automate_git_import_multiple_domains(
```

Comment:
Do you need to enable `git_owner` server role here?

Reply:
Already [enabled globally.](https://github.com/ManageIQ/integration_tests/pull/8600/files#diff-12d53d617972cfe42469893013d596d0R9)

Reply:
Oh, my bad. Didn't see it there.